app/views/categories/show.haml

```diff
@@ -0,0 +1,9 @@
+.grid-container
+  .grid-x
+    .cell.small-12.medium-3
+      = render 'shared/sidebar', categories: @categories
```

Comment:
where is `@categories` defined? I guess we also don't need it as we use `Category.all`

Reply:
Yes, you're right. Probably just a leftover from the initial approach on the sidebar I overlooked. Removed it in #17